libraries/botframework-streaming-extensions/tests/Assembler.test.js

```diff
@@ -0,0 +1,192 @@
+const Header = require('../lib/Payloads/Models/Header');

```

Comment:
why can't you not use the import syntax in all of these test files?

Reply:
This is just an artifact of the original lint rules the project followed, I'll go through and clean them up.

Reply:
Actually I'm wrong, these are matching the patterns set by the JS tests in the rest of the libraries. I have cleaned the tests up, removed ones that no longer had meaning, updated the remaining to reflect the changes to the code, and am in the process of repairing our coverage metrics now, though.